src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/returncount/Example2.java

```diff
@@ -0,0 +1,46 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="ReturnCount">
+      <property name="maxForVoid" value="0"/>
+    </module>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.returncount;
+
+// xdoc section -- start
+import java.util.function.Predicate;
+
+public class Example2 {
+    public Example2() {}
+
+    // violation below, 'max allowed for void methods/constructors/lambdas is 0'
+    public Example2(int i) { return; }
```

Comment:
In previous example, such ctor is at line 18.
Can we preserve line numbers to be same for all elements of class?

Please look at diff examples pictures in issue description.

Reply:
Done